docs/README.md

```diff
@@ -236,6 +236,17 @@ Usage
 Upgrade process
 ---------------------
 To upgrade please reference _Installation process_ section but replace `helm install` by `helm upgrade` command
+
+* Note about Upgrade
+  > There is no support at this time for upgrading or deleting CRDs using [Helm](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/).
+
+  In order to upgrade CRD use `kubectl apply -f ...` command.
+  ```bash
+  export CSI_OPERATOR_VERSION=VERSION WITH PATH
+  wget http://artifactory/csi-operator/$CSI_OPERATOR_VERSION/csi-baremetal-operator-$CSI_OPERATOR_VERSION.tgz
```

Comment:
BTW what do you mean by `artifactory`?

Reply:
we need to have instructions which can be simply copied and executed

Reply:
It's depends on requirements, there is charts placed. The goal is download charts and apply.

Reply:
Or do what we did for objectscale and make it automatic: 
https://eos2git.cec.lab.emc.com/ECS/objectscale-charts/pull/23/files

Reply:
@eddiepavkovic please don't add links to internal dell resources in public repo. Thanks!